src/writeConfluence.js

```diff
@@ -0,0 +1,58 @@
+const { markdown } = require('markdown')
+const Confluence = require('confluence-api')
+
+const config = {
+    username: process.env.CONFLUENCE_USER,
+    password: process.env.CONFLUENCE_API_KEY,
+    baseUrl:  "https://ecobee.atlassian.net/wiki"
+}
+const confluence = new Confluence(config)
+
+const CONFLUENCE_QA_SPACE = "QA"
+
+const createConfluencePageForRelease = (space, title, html, parentId) => {
+    confluence.postContent(space, title, html, parentId, (err, response) => {
+        if(err) {
+            throw new Error(response.text)
+        } 
+        return response
+    });
+}
+
+const writeConfluence = async (log, releaseDetails, repositoryName, teamName) => {
+
+    const { name, body, version } = releaseDetails
+
+    const includeTeamName = teamName ? `<p><strong>Team: ${teamName}<br /></strong></p>` : ''
+    const includeReleaseName = name ? `<p><strong>Release Name: ${name}<br /></strong></p>` : ''
+    const html = `${includeTeamName} ${includeReleaseName} <p><strong>Notes:<br /></strong></p> ${markdown.toHTML(body)}`
+
+    const today = new Date();
+    const month = `0${today.getMonth()+1}`.slice(-2)
+    const date = `${today.getFullYear()}-${month}-${today.getDate()}`
+    const title = `${date} - ${repositoryName} ${version}`
+    
+    // Find the Releases page for the current year
+    confluence.getContentByPageTitle(CONFLUENCE_QA_SPACE, `${today.getFullYear()} Releases`, (err, response) => {
```

Comment:
We can probably create a util function to get the parent page title. So it is easier to read and avoid the repetition of
```
today.getFullYear() Releases
```